src/content_script.js

```diff
@@ -58,6 +59,13 @@ function isFixed (elem) {
   return false;
 }
 
+function isDisplayNone (elem) {
+  do {
+    if (getComputedStyle(elem).getPropertyValue("display") == "none") return true;
```

Comment:
nit: use `===` for strict equality comparison.

Reply:
elsewhere we have `window.getComputedStyle` and here we just call `getComputedStyle` directly?